File: src/com/bean/Watchman.java
package com.bean;

public class Watchman {
	private int count;
	private int sal;
	public Watchman(int count, int sal) {
		super();
		this.count = count;
		this.sal = sal;
	}
	public int getCount() {
		return count;
	}
	public void setCount(int count) {
		this.count = count;
	}
	public int getSal() {
		return sal;
	}
	public void setSal(int sal) {
		this.sal = sal;
	}
	@Override
	public String toString() {
		return "Watchman [count=" + count + ", sal=" + sal + "]";
	}

	
}

File: src/com/bean/Watchman.go
package bean

import (
	"fmt"
)

type Watchman struct {
	count int
	sal   int
}

func CreateWatchman(count int, sal int) Watchman {
	return Watchman{count: count, sal: sal}
}
func (w Watchman) getCount() int {
	return w.count
}
func (w Watchman) setCount(count int) {
	w.count = count
}
func (w Watchman) getSal() int {
	return w.sal
}
func (w Watchman) setSal(sal int) {
	w.sal = sal
}
func (w Watchman) String() string {
	return fmt.Sprintf(" Watchman [count=%d, sal=%d]", w.count, w.sal)
}
